engines/sci/sound/midiparser_sci.cpp

```diff
@@ -433,7 +432,6 @@ void MidiParser_SCI::sendInitCommands() {
 			sendToDriver(0xB0 | i, 0x07, 127);	// Reset volume to maximum
 			sendToDriver(0xB0 | i, 0x0A, 64);	// Reset panning to center
 			sendToDriver(0xB0 | i, 0x40, 0);	// Reset hold pedal to none
-			sendToDriver(0xB0 | i, 0x4E, 0);	// Reset velocity to none
```

Comment:
Don't we need to do this?

Reply:
For SCI1+:
If we do this we will actually counter-act the initial mute setting that we're trying to fix here.

And I haven't found any evidence that the original interpreters do that. They seem to send controllers 7 (volume), 10 (pan), 0x7f (all notes off), 0x40 (sustain pedal), sometimes 0x7b if the channels need to be released.

Actually, if you look in `midiparser_sci.cpp, MidiParser_SCI::remapChannel(), lines 985-992` you can see what the original does. I think this code is from wjp and he got it very precise, I think.

The control 0x4E also is never sent to the device. It is simply handled inside remapChannels(): The muted channel just won't get mapped to the decice). So it wouldn't actually make sense to send it.

For SCI0:
From the comments in our code I would have assumed that this may play some role somehow somewhere. But I haven't seen any evidence for any use of this, neither in the interpreter nor in the drivers.


Actually, from Git history it seems that you are the one who added this:
https://github.com/scummvm/scummvm/commit/fa079a6d6f118a09a9bdeb678f57584fb8e1d8d4
Do you maybe remember why you did that?
We didn't even have the SCI1+ channel remapping back then, so maybe it was something that would now be obsolete.
But if you do remember, whether you did this for a particular game, we could test that...

Reply:
Thanks for bisecting this @athrxx!
Unfortunately I can't remember why I added that. It may have been a leftover from the code we had before channel remapping.
In any case, removing this is the right way to go :)